### PR TITLE
feat(sampling): show more clearly that an insight is sampled on the insight card

### PR DESCRIPTION
**Pull request by yakkomajuri**
Sometimes the sampling filter might be below the fold or otherwise not super clear.

Thus, let's show on the insight card very clearly that the result is sampled. 

Required me moving things around a bit to make everything fit on the funnels view (even tried the smallest wording I could think of - _X% sampling_). On all other views things fit just fine. 

Thoughts on the wording @lharries  and thoughts on the funnels view @Twixes? Feel free to jump in if you have minor adjustments in mind.

Screenshot 2023-03-08 at 15 29 17
Screenshot 2023-03-08 at 15 29 42

### CODE REVIEW

**Comment by lharries** [comment]
Wording is very clear and positioning is great

File: frontend/src/scenes/insights/InsightContainer.tsx [hunk 1]
```diff
@@ -202,6 +202,18 @@ export function InsightContainer({
                 className="insights-graph-container"
             >
                 <div>
+                    {isFunnelsFilter(filters) ? (
+                        <Row
```
Comment (pauldambra) [inline frontend/src/scenes/insights/InsightContainer.tsx:206]: ideally we'd avoid adding more antd and use `<div className="flex flex-col"` or `<div className="flex flex-row` as appropriate but IIRC there're loads of `Row` and `Col` in insights so two more is 🤷

**Review by pauldambra — APPROVED** [review]
agree positioning and text are 💯

File: frontend/src/scenes/insights/InsightContainer.tsx (hunk 1 above)
Comment (yakkomajuri) [reply frontend/src/scenes/insights/InsightContainer.tsx]: updated, thanks

**Review by Twixes — COMMENTED** [review]
Makes sense, but this introduced a visual regression in experiments, where there is now a double border above the funnel. (https://github.com/PostHog/posthog/pull/14636/files#diff-2dc2a02bdc863301f3109266f5a18d6cfb5e3865f10ddba00383d3816f6cd606)
And _three_ meta rows above the funnel chart seem a bit excessive. The new bar pushes the results farther below the page, requiring more scrolling. Perhaps we could just stick the exact sampling percentage in a tooltip, and only signify whether sampling was used or not with a lightning bolt icon in the existing row? Would be awesome to avoid proliferation of rows, since they make the chart _look_ more complex than it really is.

**Pull request merged**